app/views/pays/show.html.erb

```diff
@@ -37,15 +37,24 @@
                     </li>
                     <li>
                       <b><%= t ("total_pay") %></b>
-                      <%= product.price * quantity  %>
+                      <%= product.price * quantity %>
```

Comment:
Hạn chế viết logic ruby trong view nhé anh? Anh có thể dùng js để tính toán thay vì viết như này